Fang-4-Group/Graduation-Project
# Jeaninetang/gp 126/makethe  update profile

**Test**

- Fill in the Login page
![image](https://github.com/user-attachments/assets/c7c1cb8b-f57e-4e9a-910b-5d15125232af)

- Information should be correctly presented on the profile page
![image](https://github.com/user-attachments/assets/9684fdb6-50e4-4c06-8bae-bb9fdb760257)

- There are two buttons on the top right coner of the profile page

- Click on the first button on the profile page and it will redirect to the update page, the information on the update page should be exactly the same as what you have seen in your profile page
![image](https://github.com/user-attachments/assets/f23f10d9-ce17-4b9b-b518-3d8e96232c5c)

- Click on the second button on the profile page and it will lead you to the page for presenting your profile to others, which doesn't contain the update button the the top right corner
![image](https://github.com/user-attachments/assets/eff0f59f-d204-4e77-a3b4-d6fe1e4dc793)
